src/model/TissueSpeficity.tsx

```diff
@@ -0,0 +1,64 @@
+import React, { Fragment } from 'react';
+import v1 from 'uuid';
+import UniProtEvidenceTag from '../components/UniProtEvidenceTag';
+import { EvidenceType } from './types/modelTypes';
+import { FreeTextType } from './FreeText';
+
+type Comments = {
+  commentType: FreeTextType;
+  texts: [
+    {
+      value: string;
+      evidences?: EvidenceType[];
+    }
+  ];
+};
+
+type TissueData = {
+  comments?: Comments[];
+};
+
+type TissueProps = {
+  data: TissueData;
+};
+
+export const TissueSpeficity: React.FC<TissueProps> = ({ data }) => {
+  const isValid = (d: Comments) =>
+    d.commentType === FreeTextType.TISSUE_SPECIFICITY &&
+    d.texts &&
+    d.texts.length > 0;
+  if (!data.comments || !data.comments.some(d => isValid(d))) {
+    return null;
+  }
+  const tissueSpecifityData = data.comments.filter(d => isValid(d));
+  if (!tissueSpecifityData || tissueSpecifityData.length <= 0) {
```

Comment:
Same as previous comment above. I think it's better to check for a positive condition in the `length` check.

Reply:
likewise as above. changed.